src/oc-client.js

```diff
@@ -520,7 +520,11 @@ var oc = oc || {};
 
         oc.renderByHref($component.attr('href'), function (err, data) {
           if (err || !data) {
-            $component.html('');
+            $component
+              .attr('data-rendering', 'false')
+              .attr('data-rendered', 'false')
```

Comment:
won't this cause issues with `.renderAllUnrenderedComponents` functionality?

Reply:
how? explicitly setting to false or not to true doesn't make a difference, and `data-rendered` will never be true if we get here, since we don't call `processHtml`